controllers/servers/csi/addons_server.py

```diff
@@ -27,13 +27,14 @@ def EnableVolumeReplication(self, request, context):
 
         connection_info = utils.get_array_connection_info_from_secrets(request.secrets)
         with get_agent(connection_info, volume_id_info.array_type).get_mediator() as mediator:
-            volume = mediator.get_object_by_id(volume_id, servers_settings.VOLUME_TYPE_NAME)
-            if not volume:
-                raise array_errors.ObjectNotFoundError(volume_id)
+            volume, error_message = self._get_and_validate_volume(mediator, volume_id)
```

Comment:
`and` indicates that we are not keeping SOLID.. please split it

Reply:
Done